### ISSUE TITLE
Error while trying to create a Mavensmate project in Windows 7 (64 bits)

### ISSUE THREAD

**Issue by riadhmankai** [joeferraro/MavensMate-SublimeText#584]
Hello,
I'v been struggling to properly setup mavensmate in Windows. I've been a Sublime Text / Mavensmate user in Mac OS for more than a year and never had a problem. But now I have to switch to Windows 7 64 bits (imposed by my business).

The problem is that when I create a mavensmate project, after entering the login/password and clicking create, I get the following error : 

Traceback (most recent call last): File "", line 12, in File "", line 9, in main File "C:\Users\Administrator\Desktop\ghook\build\mm\out00-PYZ.pyz\mm.request", line 95, in execute File "C:\Users\Administrator\Desktop\ghook\build\mm\out00-PYZ.pyz\mm.request", line 133, in __printr File "C:\Users\Administrator\Desktop\ghook\build\mm\out00-PYZ.pyz\json", line 243, in dumps File "C:\Users\Administrator\Desktop\ghook\build\mm\out00-PYZ.pyz\json.encoder", line 207, in encode File "C:\Users\Administrator\Desktop\ghook\build\mm\out00-PYZ.pyz\json.encoder", line 270, in iterencode UnicodeDecodeError: 'utf8' codec can't decode byte 0xe9 in position 289: invalid continuation byte 

I really tried a lot of things with no result. I looked at bug n° #475 in Github, but it didn't help me. I properly installed python in Windows7. I also setup mm_python_location and mm_workspace in the user setting file.

Thank you in advance for any help.
Riadh

**Comment by jlcounio** [comment]
same issue for me

**Comment by riadhmankai** [comment]
Do anyone has a clear procedure on how to install Mavensmate on Windows 7 64 bits environment? Thanks!

**Comment by jlcounio** [comment]
i have the same issue  on windows 8.1

**Comment by riadhmankai** [comment]
Hello,
I finally solved the problem. It was a proxy issue. All I had to do is to to set up my company proxy in the Mavensmate User Settings.
Here is the list of steps that I followed in order to make Mavensmate work in Windows 7 64 bits :
- Download Sublime Text 32 bits
- Install Python 2.7
- Install Package Control in Sublime Text : https://packagecontrol.io/installation
- Install Mavensmate with Package Control : http://mavensmate.com/Plugins/Sublime_Text/Installation
- Set up theses variables in user settings file : mm_workspace, mm_http_proxy (if any), mm_https_proxy (if any), mm_python_location, and mm_python_location (you will find this one in C:/Users/YOUR_WINDOWS_USERNAME /AppData/Roaming/Sublime Text 3/Packages/User/MavensMate/mm/mm.exe
- Restart Mavensmate and enjoy you coding!
